src/models/update-many-at-once-options.models.ts

```diff
@@ -54,10 +54,14 @@ export interface UpdateManyAtOnceOptions<U> {
 	 * Defaults to true.
 	 */
 	unsetUndefined?: boolean;
-
 	/**
 	 * Return cursor on new entities.
 	 * Default: true
 	 */
 	returnNewEntities?: boolean;
+	/**
+	 * Max number of task running concurently
+	 * Default: 1
+	 */
+	poolNbMaxConcurrency?: number;
```

Comment:
one should be able to give directly a Promise Pool instance

Reply:
done!